src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -319,6 +320,18 @@ public function iShouldSeeWithQuantityInMyCart(string $productName, int $quantit
         $this->checkProductQuantity($this->cartsClient->getLastResponse(), $productName, $quantity);
     }
 
+    /**
+     * @Then I should be informed that cart items are no longer available
+     */
+    public function iShouldBeInformedThatCartItemsAreNoLongerAvailable(): void
+    {
+        $response = $this->cartsClient->getLastResponse() ?? $this->sharedStorage->get('response');
```

Comment:
Why are you getting a response from shared storage here? If I see it correctly, you are setting this response only after complete step, so it is not associated with the scenarios, that you are changing in this PR

Reply:
some steps are called on CheckoutContext, and there it is using AbstractBrowser class to make requests. I used there a sharedStorage to send objects between contexts. It is needed to make those steps work for now. But soon I will work on refactor of both of these contexts, so it won't be such a confussion.